master/getting-started/mesos/index.md

```diff
@@ -29,12 +29,7 @@ and improve reliability. See additional information on
 
 ## Tutorials
 
-#### [Docker Containerizer](tutorials/docker)
```

Comment:
I'm not sure how I feel about effectively duplicating the left hand menu here - which is basically what we're doing.

IMHO I think the overview page is a good place to distinguish between the different installation options and the mention the tutorials.... not necessary duplicate all of the links.

I wouldn't hold up merging because of this thought though - but it is something that niggles me a little about some of our docs.